lucille-core/src/main/java/com/kmwllc/lucille/connector/jdbc/DatabaseConnector.java

```diff
@@ -185,10 +185,13 @@ public void execute(Publisher publisher) throws ConnectorException {
                 String.format("Field name \"%s\" is reserved, please rename it or add it to the ignore list", fieldName));
           }
 
-          String fieldValue = rs.getString(i);
-          // log.info("Add Field {} Value {} -- Doc {}", fieldName, fieldValue, doc);
+          Object fieldValue = rs.getObject(i);
           if (fieldValue != null) {
-            doc.setOrAdd(fieldName, fieldValue);
+            try {
+              doc.setOrAdd(fieldName, fieldValue);
+            } catch (IllegalArgumentException e) {
+              log.warn("Error encountered while adding database object to Lucille document: {}");

```

Comment:
seems to be missing an argument to bind to the "{}" -- also should include e as last argument so the log event includes the full stacktrace